source/agora/utils/Log.d

```diff
@@ -156,7 +164,7 @@ public class CircularAppender : Appender
     {
         // edge-case: if the buffer isn't filled yet,
         // write from buffer index 0, not the cycle's current index
-        if (this.used_length <= this.buffer.length)
```

Comment:
this was the actual bug, used length is ALWAYS <+ this.buffer.length, so the else branch never executed

the this.used_length == this.buffer.length case is handled exactly same for the if and the else branch